airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigRepository.java

```diff
@@ -206,6 +207,8 @@ public List<StandardSourceDefinition> listStandardSourceDefinitions(final boolea
     final List<StandardSourceDefinition> sourceDefinitions = new ArrayList<>();
     for (final StandardSourceDefinition sourceDefinition : persistence.listConfigs(ConfigSchema.STANDARD_SOURCE_DEFINITION,
         StandardSourceDefinition.class)) {
+      sourceDefinition.withProtocolVersion(AirbyteProtocolVersion
```

Comment:
should this be change to a common method or something like:

return persistence.listConfigs(ConfigSchema.STANDARD_SOURCE_DEFINITION,
        StandardSourceDefinition.class)
.map(extract the protocol version)
.filter(filter the tombstone)

Reply:
Nit, we should use set since we don't use the returned value of `withProtocolVersion`